templates/hooks/certificate_authority/sdk_update_pre_build_request.go.tpl

```diff
@@ -0,0 +1,7 @@
+    latest, err = rm.sdkFind(ctx, latest)
+    if delta.DifferentAt("Spec.Tags") {
+		err := rm.syncTags(ctx, desired, latest)
+		if err != nil {
+			return nil, err
+		}
+	}
```

Comment:
We also need to add
```
if !delta.DifferentExcept("Spec.Tags") {
    return desired, nil
}
```
Which will basically avoid triggering a full update if a user modifies only the tags